css-paint-api/Overview.bs

```diff
@@ -138,13 +138,22 @@ interface PaintWorkletGlobalScope : WorkletGlobalScope {
 };
 </pre>
 
+The {{PaintRenderingContext2DSettings}} allows the settings for the rendering context associated
```

Comment:
This sentence is confusing, what do you mean "allows the settings"? Maybe contains or provides?

Reply:
Done